### ISSUE TITLE
Add citation narrative to the sunpy core docs

### ISSUE THREAD

**Issue by dstansby** [sunpy/sunpy#6378]
### Provide a general description of the issue or problem.

As part of the effort to make sunpy.org tailored towards the project as opposed to the core package, we should move https://sunpy.org/about#acknowledging-or-citing-sunpy to the sunpy core documentation as it is specifically about citing sunpy core. The source file lives here: https://github.com/sunpy/sunpy/blob/main/sunpy/CITATION.rst

We should embed these instructions somewhere in the sunpy core docs.

**Comment by nabobalis** [comment]
We have this on the sidebar of the main docs. https://docs.sunpy.org/en/stable/about.html 

Is there more to be done?

**Comment by ayshih** [comment]
Agreed that nothing needs to be done in the core docs.  The website needs to be updated to describe how to acknowledge the SunPy Project and then point to the page in the core docs for how to acknowledge the core library.
